job_definitions/index_briefs.yml

```diff
@@ -0,0 +1,44 @@
+{% for environment in ['preview'] %}
+- job:
+    name: "index-briefs-{{ environment }}"
+    display-name: "Index briefs - {{ environment }}"
+    project-type: freestyle
+    description: "This imports all briefs (opportunities) in the current API database into the current Elasticsearch index."
+    parameters:
+      - string:
+          name: INDEX
+          default: {{ search_config[environment].default_index }}
+          description: "Index name or alias to use (eg 'briefs-digital-outcomes-and-specialists-2')"
```

Comment:
We dropped the `-2` off the end of the index name

Reply:
Guess I'm not up-to-date on this; is the decision, then, that multiple framework iterations are potentially going to be stored in a single index under different document names?

Reply:
They definitely will in the case of DOS - the existing list of opportunities includes (if you choose to search for closed briefs) the DOS-1 briefs, too.